### ISSUE TITLE
socket units not supported

### ISSUE THREAD

**Issue by Jamie-sas** [gdraheim/docker-systemctl-replacement#30]
One feature of systemd is to allow a [systemd.socket](https://www.freedesktop.org/software/systemd/man/systemd.socket.html) file to be created in parallel with a .service file.

In other words, you might have

foo.service
foo.socket

In the same directory.

These .socket files are apparently unsupported by systemctl.py.

Note also that they can have the same %t that is currently not supported in .service files, as reported in issue #29.

**Comment by gdraheim** [comment]
I did never have a need to support sockets. 

Patches may need to change a lot of things as only *.service files are scanned. And those have a lmited concept of starting other services before.

**Comment by gdraheim** [comment]
I am going to set this to wontfix by now. May be you have an implementation idea how to actualy do it in a script like systemctl.py ?

**Comment by gdraheim** [comment]
I have put the basic infrastructure into place - however Accept=no will simply start the underlying service unit. An actual port-listen, required for Accept=yes, is not attempted. But that's the original use case?
